docs/user/sbt.md

```diff
@@ -126,25 +119,16 @@ nativeConfig ~= { c =>
 | 0.1    | `package`               | `File`          | Similar to standard package with addition of NIR                              |
 | 0.1    | `publish`               | `Unit`          | Similar to standard publish with addition of NIR (1)                          |
 | 0.1    | `nativeLink`            | `File`          | Link NIR and generate native binary                                           |
-| 0.1    | `nativeClang`           | `File`          | Path to `clang` command                                                       |
-| 0.1    | `nativeClangPP`         | `File`          | Path to `clang++` command                                                     |
-| 0.1    | `nativeCompileOptions`  | `Seq[String]`   | Extra options passed to clang verbatim during compilation                     |
-| 0.1    | `nativeLinkingOptions`  | `Seq[String]`   | Extra options passed to clang verbatim during linking                         |
-| 0.1    | `nativeMode`            | `String`        | One of `"debug"`, `"release-fast"`, `"release-size"` or `"release-full"` (2)  |
-| 0.2    | `nativeGC`              | `String`        | One of `"none"`, `"boehm"`, `"immix"` or `"commix"` (3)                       |
-| 0.3.3  | `nativeLinkStubs`       | `Boolean`       | Whether to link `@stub` definitions, or to ignore them                        |
 | 0.4.0  | `nativeConfig`          | `NativeConfig`  | Configuration of the Scala Native plugin                                      |
-| 0.4.0  | `nativeLTO`             | `String`        | One of `"none"`, `"full"` or `"thin"` (4)                                     |
-| 0.4.0  | `targetTriple`          | `String`        | The platform LLVM target triple                                               |
-| 0.4.0  | `nativeCheck`           | `Boolean`       | Shall the linker check intermediate results for correctness?                  |
-| 0.4.0  | `nativeDump`            | `Boolean`       | Shall the linker dump intermediate results to disk?                           |
+| 0.5.0  | `nativeLinkReleaseFast` | `File`          | Alias for `nativeLink` using fast release build mode                          |
+| 0.5.0  | `nativeLinkReleaseFull` | `File`          | Alias for `nativeLink` using full release build mode                          |
+
+For the details of available `NativeConfig` options see [API](https://javadoc.io/doc/org.scala-native/tools_3/latest/scala/scalanative/build/NativeConfig.html)
 
-1.  See [Publishing](#publishing) and [Cross
-    compilation](#cross-compilation) for details.
+1.  See [Publishing](#publishing) and [Cross compilation](#cross-compilation) for details.
 2.  See [Compilation modes](#compilation-modes) for details.
 3.  See [Garbage collectors](#garbage-collectors) for details.
-4.  See [Link-Time Optimization (LTO)](#link-time-optimization-lto) for
-    details.
+4.  See [Link-Time Optimization (LTO)](#link-time-optimization-lto) for details.
```

Comment:
I think points 2, 3 and 4 are not referenced anywhere anymore, so leaving this here (at least formatted like this) could be a bit confusing.